manual/development-process/applying-or-destroying-a-bastion-host.md

```diff
@@ -0,0 +1,20 @@
+# Applying or destroying a Bastion host
+
+## Applying a Bastion host
```

Comment:
Thanks for adding this. Could you add a line saying why you would want to create a Bastion? And add links between the two pages?

We should also link to these pages from the index in the manual: https://github.com/nationalarchives/tdr-dev-documentation/tree/master/manual#developer-manual

Reply:
We should also add a note saying something like:

> A bastion should only ever be created as a last resort in order to debug something that we've failed to debug using other methods like application logs. Because it provides a way of connecting to the database, the bastion **must** be removed as soon as you have finished investigating the problem.

Reply:
It isn't only for this though, @pashworth-tna  uses it for his work. Arguably in the future we'll have some sort of replicated data warehouse that he can use instead but for now, as far as I'm aware, he's the main user.

Reply:
Yep, but even that's supposed to be a last resort! We'll be building API endpoints for him to get most of the data he needs. Connecting to the real DB will only if he needs to urgently get some data that we haven't added to the analytics API endpoints yet.